src/saleor_app/endpoints.py

```diff
@@ -59,25 +56,17 @@ async def install(
 
 async def handle_webhook(
     request: Request,
-    payload: List[Any],  # FIXME provide a way to proper define payload types
-    _domain_is_valid=Depends(verify_saleor_domain),
+    payload: List[Webhook],  # FIXME provide a way to proper define payload types
     saleor_domain=Depends(saleor_domain_header),
-    event_type=Depends(webhook_event_type),
-    _signature_is_valid=Depends(verify_webhook_signature),
+    _event_type=Header(None, alias=SALEOR_EVENT_HEADER),
 ):
-    response = {}
-    handler = request.app.extra["saleor"]["webhook_handlers"].get(event_type)
-    if handler is not None:
-        response = await handler(payload, saleor_domain)
-    return response or {}
+    return {}
 
 
 async def get_public_form(commons: ConfigurationFormDeps = Depends()):
     context = {
-        "request": commons.request,
-        "form_url": commons.request.url,
-        "domain": commons.saleor_domain,
+        "request": str(commons.request),
+        "form_url": str(commons.request.url),
+        "saleor_domain": commons.saleor_domain,
     }
-    return Jinja2Templates(directory=str(commons.settings.static_dir)).TemplateResponse(
-        "configuration/index.html", context
-    )
+    return PlainTextResponse(json.dumps(context, indent=4))
```

Comment:
As stated above, I wanted to get rid of Jinja as a dependency

Reply:
So maybe we should fully drop this function? I mean, maybe this is something which should be explicitly defined in example Apps (with Jinja as an example).